8a3fde0.diff

```diff
@@ -0,0 +1,20 @@
+diff --git a/src/corelib/Qt5CoreMacros.cmake b/src/corelib/Qt5CoreMacros.cmake
```

Comment:
is this diff the same as this change https://codereview.qt-project.org/c/qt/qtbase/+/288359 ?
if yes, it would be great if you could include the commit message in the diff, that would give us the Change-Id, and qt task number, so anyone can understand why is this patch here

Reply:
Done :)